docker-compose.yml

```diff
@@ -7,6 +7,9 @@ x-base_service: &base_service
       - &v1 ./data:/data
       - &v2 ./output:/output
     stop_signal: SIGINT
+    security_opt:
+      - label=type:nvidia_container_t
+    runtime: nvidia
```

Comment:
what would happen here if the machine does not have a gpu?

Reply:
Fantastic question! I have another machine I can test with that doesn't have an nvidia gpu, I'll report back later today when I get some time to test. My initial thought is to move this statement to the gpu only ones if it explodes.

Reply:
@AbdBarho Yes, thank you for having me check this! Podman on my non-nvidia tablet complains that it's missing a security context label, so it shouldn't be here by default. Perhaps it's best left commented out? I'm fine with removing it from here and adding it to documentation too.

I didn't try it with Docker on the tablet, but the fact that I'm having issues with Podman makes me think it would also have issues.

Reply:
maybe we can use yaml anchors just like the base service.

```yml
x-gpu-config: &gpu_config
   security_opt:
      - label=type:nvidia_container_t
    runtime: nvidia


services:
  auto:
    <<: *base_service
    <<: *gpu_config
    # ...
```

I should also test this setup as well.

Reply:
@AbdBarho I could do that too. I'd have to make auto-cpu be the "base" and then make auto extend it, otherwise both will get the label. Should I comment out the `<<: *gpu_config` and let people with systems that need to un-comment it?

Reply:
Unfortunately, many users of this repo are not very tech savy, and I prefer to not have them fiddle around with the files

but I agree what I wrote is incorrect, this label should only exist on the GPU one...

Reply:
I added the gpu_config. Please do test!

Reply:
That does not seem to work, at least on windows:
```
Error response from daemon: Unknown runtime specified nvidia
```

maybe we can move this problem to another issue / pr so we can finish this.